controller/search_blackbox_test.go

```diff
@@ -492,21 +493,21 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 
 	s.T().Run("space=ID AND label=REST : expect 0 itmes", func(t *testing.T) {
 		filter := fmt.Sprintf(`
-				{"$AND": [
-					{"space":"%s"},
-					{"label": "%s"}
-				]}`,
+						{"$AND": [
+							{"space":"%s"},
+							{"label": "%s"}
+						]}`,
 			spaceIDStr, fxt.LabelByName("rest").ID)
 		_, result := test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
 		assert.Len(t, result.Data, 0) // no items having REST label
 	})
 
 	s.T().Run("space=ID AND label != Backend", func(t *testing.T) {
 		filter := fmt.Sprintf(`
-				{"$AND": [
-					{"space":"%s"},
-					{"label": "%s", "negate": true}
-				]}`,
+						{"$AND": [
```

Comment:
What is the point with all these format changes? I don't grok it.

Reply:
My editor does that by default.